code/game/gamemodes/handofgod/structures.dm

```diff
@@ -0,0 +1,591 @@
+			GG.use(spend)
+			check_completion()
+		else
+			user << "<span class='notice'>\The [src] does not require any more greater gems!"
+		return
+
+	..()
+
+
+/obj/structure/divine/construction_holder/proc/check_completion()
+	if(!metal_cost && !glass_cost && !lesser_gem_cost && !greater_gem_cost)
+		visible_message("<span class='notice'>\The [initial(construction_result.name)] is complete!</span>")
+		var/obj/structure/divine/D = new construction_result (get_turf(src))
+		D.assign_deity(deity)
+		qdel(src)
+
+
+/obj/structure/divine/construction_holder/examine(mob/user)
+	..()
+
+	if(metal_cost || glass_cost || lesser_gem_cost || greater_gem_cost)
+		user << "To finish construction it requires the following materials:"
+		if(metal_cost)
+			user << "[metal_cost] metal <IMG CLASS=icon SRC=icons/obj/items.dmi ICONSTATE='sheet-metal'>"
+		if(glass_cost)
+			user << "[glass_cost] glass <IMG CLASS=icon SRC=icons/obj/items.dmi ICONSTATE='sheet-glass'>"
+		if(lesser_gem_cost)
+			user << "[lesser_gem_cost] lesser gems <IMG CLASS=icon SRC=icons/obj/items.dmi ICONSTATE='sheet-lessergem'>"
+		if(greater_gem_cost)
+			user << "[greater_gem_cost] greater gems <IMG CLASS=icon SRC=icons/obj/items.dmi ICONSTATE='sheet-greatergem'>"
+
+
+/obj/structure/divine/nexus
+	name = "nexus"
+	desc = "It anchors a deity to this world. It radiates an unusual aura. Cultists protect this at all costs. It looks well protected from explosive shock."
+	icon_state = "nexus"
+	health = 500
+	maxhealth = 500
+	constructable = FALSE
+	var/faith_regen_rate = 1
+	var/list/powerpylons = list()
+
+
+/obj/structure/divine/nexus/ex_act()
+	return
+
+
+/obj/structure/divine/nexus/healthcheck()
+	if(deity)
+		deity.update_health_hud()
+
+	if(!health)
+		if(!qdeleted(deity) && deity.nexus_required)
+			deity << "<span class='danger'>Your nexus was destroyed. You feel yourself fading...</span>"
+			qdel(deity)
+		visible_message("<span class='danger'>\The [src] was destroyed!</span>")
+		qdel(src)
+
+
+/obj/structure/divine/nexus/New()
+	SSobj.processing |= src
+
+
+/obj/structure/divine/nexus/process()
+	healthcheck()
+	if(deity)
+		deity.update_followers()
+		deity.add_faith(faith_regen_rate + (powerpylons.len / 5) + (deity.alive_followers / 3))
+		deity.max_faith = initial(deity.max_faith) + (deity.alive_followers*10) //10 followers = 100 max faith, so disaster() at around 20 followers
+		deity.check_death()
+
+
+/obj/structure/divine/nexus/Destroy()
+	SSobj.processing -= src
+	return ..()
+
+
+/obj/structure/divine/conduit
+	name = "conduit"
+	desc = "It allows a deity to extend their reach.  Their powers are just as potent near a conduit as a nexus."
+	icon_state = "conduit"
+	health = 150
+	maxhealth = 150
+	metal_cost = 20
+	glass_cost = 5
+
+/obj/structure/divine/conduit/assign_deity(mob/camera/god/new_deity, alert_old_deity = TRUE)
+	if(deity)
+		deity.conduits -= src
+	..()
+	if(deity)
+		deity.conduits += src
+
+/* //No good sprites, and not enough items to make it viable yet
```

Comment:
"No good sprites" we can easily fix that, and for items,it shouldn't be too hard!